openmls/src/schedule/pprf/mod.rs

```diff
@@ -0,0 +1,316 @@
+use std::collections::HashMap;
```

Comment:
I think this module needs a bit of design documentation. Since this is not part of the draft, could you describe a little what you're implementing here?